engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -246,6 +249,7 @@ public void onOpened() {
             worldsDropdown.setOptions(worldNames());
         }
         selectedWorld = "";
+        indexOfWorld =findIndex(worlds, selectedWorld);
```

Comment:
Trivial: Missing whitespace. Be sure to check the affected files with https://github.com/MovingBlocks/Terasology/wiki/Checkstyle - would have caught this one :-)